src/coreclr/jit/emitwasm.cpp

```diff
@@ -191,16 +242,40 @@ size_t emitter::emitOutputInstr(insGroup* ig, instrDesc* id, BYTE** dp)
             break;
         case IF_BLOCK:
             dst += emitOutputByte(dst, opcode);
-            dst += emitOutputByte(dst, 0x40);
+            dst += emitOutputByte(dst, 0x40 /* block type of void */);
             break;
         case IF_ULEB128:
+        case IF_LEB128:
+        {
             dst += emitOutputByte(dst, opcode);
-            // TODO-WASM: emit uleb128
+            cnsval_ssize_t constant = emitGetInsSC(id);
+            dst += emitEncodeLEB64(dst, &constant, (insFmt == IF_LEB128));
             break;
+        }
+        case IF_F32:
+        case IF_F64:
+        {
+            dst += emitOutputByte(dst, opcode);
+            /* FIXME: How do I get the constant from inside the emitter? */
```

Comment:
Codegen for float/dbl constants should create an instrDesc (of suitable type, we may need a new kind) with room for the appropriate value, and then you can add an accessor to call here.